features.yaml

```diff
@@ -1,299 +1,344 @@
 features:
   - name: "Protocols:HTTP1.1/HTTP2/gRPC/TCP"
+    id: 1
```

Comment:
What would you think of making the id meaningful to humans as well?  something like basic-protocols?

Then, when we want to refer to this feature elsewhere (for instance in @zhlsunshine's API maturity annotations), the reference is slightly readable, where an integer is not.

Reply:
Sure. I'm fine with that. Will the API maturity annotations look up the feature name? The name is rendered to Istio.io so may be useful for consistency in things like istioctl.